docs/static/ds6/skin.css

```diff
@@ -2114,57 +2114,115 @@ span.inline-notice {
-  height: 40px;
+  background-position: right 50%;
+  pointer-events: none;
+  position: absolute;
+  right: 16px;
+  top: 0;
+}
+svg.select__icon {
+  display: inline-block;
+  fill: currentColor;
+  height: 100%;
+  stroke: currentColor;
+  stroke-width: 0;
+  vertical-align: middle;
   width: 9px;
-  background-position: right calc(50% - 1px);
-  content: "";
-  margin-left: 8px;
   pointer-events: none;
   position: absolute;
   right: 16px;
-  top: 1px;
+  top: 0;
+}
+.select.fluid,
+.select.select--fluid {
+  width: 100%;
+}
+.select.fluid > select,
+.select.select--fluid > select {
+  width: 100%;
+}
+.select.select--borderless > select {
+  background-color: transparent;
+  border-color: transparent;
+}
+.select.select--borderless > select:focus {
+  border: 1px solid;
+}
+.select__icon:empty {
+  background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMjEuNiIgaGVpZ2h0PSIxMi41OCIgdmlld0JveD0iMS4zNSA1LjcgMjEuNiAxMi41OCI+PHBhdGggZD0iTTEyLjE4NiAxOC4yODVjLS40NTEtLjAwOS0uODA5LS4xNjctMS4wNzUtLjQ0MWwtOS4zMzctOS42YTEuNTI3IDEuNTI3IDAgMCAxLS40MjQtLjk5OXYtLjEwOGMuMDE1LS4zODYuMTY2LS43NDEuNDI0LTEuMDA4LjU2LS41NzMgMS41MjktLjU3IDIuMDgyIDBsOC4yOTQgOC41MyA4LjI5Mi04LjUzMmMuNTU4LS41NyAxLjUyNi0uNTcgMi4wOCAwIC4yNjUuMjcuNDE2LjYyOS40MjggMS4wMXYuMDg3Yy0uMDEyLjM5MS0uMTY1Ljc1LS40MjcgMS4wMmwtOS4zMzMgOS42YTEuNDQzIDEuNDQzIDAgMCAxLTEuMDA0LjQ0MSIvPjwvc3ZnPg==');
+  height: 100%;
+  width: 9px;
 }
 @media screen and (-ms-high-contrast: white-on-black) {
   .select::after {
     background-image: url('data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMSIgaGVpZ2h0PSIxMiI+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE5LjAwNi4zNTFMMTAuNSA5LjEwNCAxLjk5Mi4zNWExLjE0OSAxLjE0OSAwIDAgMC0xLjY1MSAwQTEuMjI2IDEuMjI2IDAgMCAwIDAgMS4xNTJ2LjA5NmMuMDEyLjI5MS4xMjYuNTc5LjM0LjgwMWw5LjMzNSA5LjZjLjIyLjIyNy41MDguMzQ0Ljc5Ni4zNTFoLjA1N2MuMjg4LS4wMDguNTc2LS4xMjQuNzk3LS4zNTFsOS4zMzMtOS42YTEuMjEgMS4yMSAwIDAgMCAuMzQyLS44MXYtLjA3OGExLjIxIDEuMjEgMCAwIDAtLjM0Mi0uODEgMS4xNDggMS4xNDggMCAwIDAtMS42NTIgMHoiLz48L3N2Zz4=');
   }
 }
 .select > select {
-  -webkit-appearance: none;
-     -moz-appearance: none;
-          appearance: none;
   background-color: #fff;
-  border: 1px solid #c7c7c7;
+  border-color: #c7c7c7;
   border-radius: 0;
   font-family: "Market Sans", Arial, sans-serif;
-  font-size: 16px;
-  height: 40px;
-  margin: 0;
+  font-size: 1rem;
```

Comment:
I'm a bit nervous about the property changes, as they got lost in the diff and I can't compare them all. I would try and minimize refactoring of properties and save them for a separate PR where they are easier to diff.  I do know we want to keep select at 16px for now to match button and textbox (which are also hardcoded).

Make sure that you update and use the existing test page. There should be no regressions (right now it's in a broken state):

<img width="897" alt="screen shot 2018-07-16 at 9 06 30 pm" src="https://user-images.githubusercontent.com/38065/42796014-35ff1150-893c-11e8-9ebc-4d6318d9ad5c.png">

Compare with:

<img width="958" alt="screen shot 2018-07-16 at 9 08 10 pm" src="https://user-images.githubusercontent.com/38065/42796046-5dddf772-893c-11e8-8153-ca46858d27b8.png">

Reply:
I've fixed the test pages.

Also, I'm looking at why that property changed. It was not intentional as I was aiming to keep all the properties the same. In the shuffle this one might have been updated.

Reply:
It got changed to this: `font-size: @font-size-16;`. I've changed it back to a hardcoded `16px`.

Reply:
Ok great. We should be good if it was your aim to keep them the same.